information-service/src/main/java/com/lms/informationservice/controller/InformationController.java

```diff
@@ -56,6 +56,20 @@ public ResponseEntity<List<Team>> fetchTeams() {
         return ResponseEntity.ok(teams);
     }
 
+    /**
+     * Endpoint to get all team data from the database and provide it to the user
+     * It differs from the fetch endpoint in that it does not interact with the API, only the database.
+     * This'll be called whenever a user wants to see information about teams in the league etc
+     * while the fetch method will be called to update the teams ranking at the start of and during the course of a league
+     *
+     *
+     * @return ResponseEntity<List<Teams>> Response entity containing a list of all teams in the database.
+     */
+    @GetMapping("/teams/get-teams")
+    public ResponseEntity<List<Team>> getTeams(){
+        List<Team> teams = db.getTeamsFromDB();
+        return ResponseEntity.ok(teams);
+    }
     /**
```

Comment:
I'm not sure what this will do. It gets all team data but is it the league table? Matches? If it's league table I don't think we need it. Only data we need with teams are matches each week (we said gonna run that every day) and results of this week matches 
.

Reply:
However, does LGTM overall just slightly confused on that, approved

Reply:
essentially it's just a way for a user or admin to get a list of all teams without us having to make an API call. It can be used if the user wants to see an overview of ALL teams in the league, for example if a lastmanstanding game had a /getInformation endpoint that just displayed general information about the league, it would take data from this endpoint. It was moreso just a "just in case we need it" for any new use cases we decide on , and a way for me to verify how to pull data from the database, not just add to it

Reply:
also, there are some changes in this PR related to changing how db urls are stored and some other minor bug fixes, so i think it should be merged anyways, if we don't need the endpoint in the end then so be it